module/merintr/inventry.c

```diff
@@ -682,7 +683,14 @@ void InventoryLButtonUp(HWND hwnd, int x, int y, UINT keyFlags)
 
    // See if mouse pointer is in main graphics area
    if (!MouseToRoom(&temp_x, &temp_y))
+    {
+      // Drop in inventory. Check if we're still in inventory first.
+      if (x < 0 || y < 0)
```

Comment:
This looks a little fishy.  No check for maximum x or y values?  Does this only work because the inventory is in the lower right corner of the window?

Reply:
Added additional checks including one that verifies that when the user lets go of the left mouse button, they are within the inventory window area per InventoryGetArea() and IsInArea() functions